tools/querytee/proxy.go

```diff
@@ -40,7 +39,8 @@ type ProxyConfig struct {
 }
 
 func (cfg *ProxyConfig) RegisterFlags(f *flag.FlagSet) {
-	f.IntVar(&cfg.ServerServicePort, "server.service-port", 80, "The port where the query-tee service listens to.")
+	f.IntVar(&cfg.ServerServicePort, "server.service-port", 80, "The HTTP port where the query-tee service listens to.")
```

Comment:
I would do an extra effort and rename this to `server.http-service-port` to clarify it.